critiquebrainz/frontend/views/review.py

```diff
@@ -226,12 +226,15 @@ def create():
         return redirect(url_for('user.reviews', user_id=current_user.id))
 
     # Checking if the user already wrote a review for this entity
-    reviews, count = db_review.list_reviews(user_id=current_user.id, entity_id=entity_id)
+    reviews, count = db_review.list_reviews(user_id=current_user.id, entity_id=entity_id, inc_drafts=True)
```

Comment:
What about hidden reviews here? the default of `inc_hidden` is still False.

Reply:
Yeah, right. I was wondering how we should handle those. If I am right, hidden reviews are reviews that are not public but visible to the author. But since we are using the user id, the hidden review will be from the user so we probably redirect the user to their review ? Does that make sense to do ?

Reply:
yes, redirecting to the review with a message sounds like a good idea.
You should work out what this flag means. Is it set if someone flags a review as bad, or is that something else? Who can set a review as hidden? An admin, or only the author?

Reply:
Here is what I figured out. Moderators can hide reviews. Any user including the author and excluding moderators gets a 403 error if they try to access a hidden review.